lib/pug.js

```diff
@@ -1,11 +1,25 @@
 'use strict';
 
+const path = require('path');
 const pug = require('pug');
 
+const configPath = path.join(process.cwd(), 'pug.config');
+const defaultConfig = { compile: {} }; // avoids key errors
+
+let config;
+try {
+  // require might fail
+  config = { ...defaultConfig, ...require(configPath) };
+} catch {
```

Comment:
If there are any errors within `pug.config.js`, they will all be muted. I would prefer we try `require.resolve(configPath)` instead. If it throws `MODULE_NOT_FOUND` then we proceed with default config, otherwise we merge `configPath`.